### PR TITLE
add call_misc_tests GH workflow

### PR DESCRIPTION
**Pull request by oshi97**
Also fixes a bug in the translation test bash script, where a non-zero exit code would only be returned
if the LAST .po file in the for loop had a msgcmp error.

J=SLAP-1815
TEST=manual,auto

tested using https://github.com/yext/answers-search-ui/pull/1659
tests run in a support branch
tests fail if any of the translations in a pot file are missing, in either the last po file or a different one
short circuits if pot file is not up to date

### CODE REVIEW

**Review by tmeyer2115 — APPROVED** [review]

File: .github/run_translation_verification.sh [hunk 1]
```diff
@@ -16,14 +16,20 @@ fi
...
+if [[ $GITHUB_REF_NAME == release/*
+  || $GITHUB_REF_NAME == hotfix/*
+  || $GITHUB_REF_NAME == master
+  || $GITHUB_REF_NAME == support/* ]]
 then
   for po_file in *.po
-    do msgcmp $po_file messages.pot
+  do
+    msgcmp $po_file messages.pot
+    exit_code=$(( exit_code + $? ))
```
Comment (yen-tt) [inline .github/run_translation_verification.sh]: Can this be `msgcmp $po_file messages.pot || exit_code=1` instead incrementing the exit code this way?

File: tests/acceptance/blocks/filtersearchcomponent.js [hunk 2]
```diff
@@ -23,6 +23,6 @@ export default class FilterSearchComponentBlock {
       .typeText(this._input, displayValue)
       .pressKey('space')
       .wait(500)
-      .click(this._selector.find('.js-yext-autocomplete-option').withExactText(displayValue));
```
Comment (yen-tt) [inline tests/acceptance/blocks/filtersearchcomponent.js]: How come the text from clicking the option does not match the option's exact display value?

File: .github/run_translation_verification.sh (hunk 1 above)
Comment (oshi97) [reply .github/run_translation_verification.sh]: yes, updated! I thought it might be nice to have the exit code reflect how many po files had errors, but that's probably not standard exit code behavior (though from reading online it sounds like exit codes in general are a little fuzzy)

File: tests/acceptance/blocks/filtersearchcomponent.js (hunk 2 above)
Comment (oshi97) [reply tests/acceptance/blocks/filtersearchcomponent.js]: think it was just a spurious failure, our current usages should always be able to match exactly. I changed it from `withText` to `withExactText` in my PR right before this

Comment (oshi97) [reply tests/acceptance/blocks/filtersearchcomponent.js]: if this pops up frequently we can try bumping the selector timeout, since this was a headless acceptance failure we don't have the browserstack video to see what happened during this run

**Review by yen-tt — APPROVED** [review]
